content/patterns/button/examples/button_idl.html

```diff
@@ -200,16 +222,17 @@ <h2>JavaScript and CSS Source Code</h2>
       </section>
 
       <section class="example-code">
-        <h2 id="sc1_label">HTML Source Code</h2>
-        <p id="sc1_description">To copy the following HTML code, please open it in CodePen.</p>
-        <div role="separator" id="sc1_start_sep" aria-labelledby="sc1_start_sep sc1_label" aria-label="Start of"></div>
+        <h2 id="sc1_label">HTML のソースコード</h2>
+        <p id="sc1_description">以下の HTML コードをコピーするには、CodePen で開いてください。</p>
+        <div role="separator" id="sc1_start_sep" aria-labelledby="sc1_start_sep sc1_label" aria-label="の開始"></div>
```

Comment:
`aria-labelledby` で自己を参照したときに `aria-label` が使用されるテクニックを使って、 `role="separator"` に `Start of HTML Source Code` のアクセシブルネームを付けているっぽい。つまり、 `HTML のソースコード の開始` とするためには、id順を入れ替える必要があるはず（HTMLの状態を見てないですがたぶんそう）


```suggestion
        <div role="separator" id="sc1_start_sep" aria-labelledby="sc1_label sc1_start_sep" aria-label="の開始"></div>
```

Reply:
あってました

<img width="1247" height="520" alt="separatorの部分について、ブラウザの表示にAccessibility Visualizerが「Start of HTML Source Code」のアクセシブルネームを示している部分と、Firefoxのアクセシビリティ インスペクタが同様のアクセシブルネームを示している部分を映りこませたスクリーンショット" src="https://github.com/user-attachments/assets/b0d86b48-3901-4f18-a636-c6b00daec74a" />